reference/configuration/doctrine.rst

```diff
@@ -499,4 +499,135 @@ can be placed directly under ``doctrine.orm`` config level.
 This shortened version is commonly used in other documentation sections.
 Keep in mind that you can't use both syntaxes at the same time.
 
+Custom Mapping
+--------------
+
+It's possible to not rely on the auto mapping and set all the configuration
+on your own.
+
+For example, you can overwrite the configuration directory for a given bundle.
+Keep in mind that the bundle you want to overwrite has to exist.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+    
+        doctrine:
+            # ...
+            orm:
+                # ...
+                auto_mapping: true
+                mappings:
+                    # ...
+                    AppBundle:
+                        dir: Resources/config/doctrine
```

Comment:
is this pointing to `Resources` inside of AppBundle? Or is this pointing at the `app/Resources....` directory? If it's the latter (I was *thinking* it was), I think we should use the `%kernel.root_dir%`.

Also, is there significance to the key being `AppBundle`? What `prefix` and `alias` will be used here?